cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ApplicationCredentialInputDTO.java

```diff
@@ -0,0 +1,30 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+public class ApplicationCredentialInputDTO extends ApplicationCredentialDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)